dashboard/lib/certificate_image.rb

```diff
@@ -276,8 +276,9 @@ def self.create_course_certificate_image(name, course = nil, donor_name = nil, c
 
       if template_file == 'self_paced_pl_certificate.png'
         total_minutes = unit_or_unit_group&.duration_in_minutes || 0
-        total_hours = (total_minutes / 60).floor
-        apply_text(image, total_hours.to_s, 30, 'Times bold', 'rgb(87,87,87)', -248, 124, 80, 30)
+        total_hours_to_half_hour = (total_minutes / 30).round / 2.0
+        hours_string = format('%<duration>g', duration: total_hours_to_half_hour)
+        apply_text(image, hours_string, 30, 'Times bold', 'rgb(87,87,87)', -248, 124, 80, 30)
       end
```

Comment:
Because the merge from staging was a doozy, I'm attaching a screenshot of the diff that was approved. I also checked locally after merging and everything still looks good.

<img width="1048" alt="Screenshot 2024-02-28 at 2 26 14 PM" src="https://github.com/code-dot-org/code-dot-org/assets/46464143/6759e480-3a4f-46c4-b2b6-d7fd5fc010c0">